src/steamship/agents/service/agent_service.py

```diff
@@ -2,15 +2,24 @@
 import uuid
 from typing import List, Optional
 
-from steamship import Block, SteamshipError, Task
+from pydantic.main import BaseModel
+
+from steamship import Block, File, SteamshipError, Task
 from steamship.agents.llms.openai import OpenAI
 from steamship.agents.logging import AgentLogging, StreamingOpts
 from steamship.agents.schema import Action, Agent, FinishAction
 from steamship.agents.schema.context import AgentContext, Metadata
 from steamship.agents.utils import with_llm
+from steamship.data import TagKind
+from steamship.data.tags.tag_constants import ChatTag
 from steamship.invocable import PackageService, post
 
 
+class StreamingResponse(BaseModel):
```

Comment:
@douglas-reid if you extend `BaseModel` instead of `CamelModel` then the serialization is underscored rather than the camelcase that we use for engine comms.

It sneaks through in the case since it's internal to an object's response, but it breaks the parsing of the task in the Typescript client.

Would it be possible to add a test that, e.g., `response.task.requestId` is serialized as `requestId` instead of `request_id` just to make sure that the clients demarshalling it won't explode?

Reply:
sure